src/os/macos.zig

```diff
@@ -110,9 +90,52 @@ pub const NSOperatingSystemVersion = extern struct {
 };
 
 pub const NSSearchPathDirectory = enum(c_ulong) {
+    NSCachesDirectory = 13,
     NSApplicationSupportDirectory = 14,
 };
 
 pub const NSSearchPathDomainMask = enum(c_ulong) {
     NSUserDomainMask = 1,
 };
+
+fn getSystemPath(
```

Comment:
A little bit of the name game, but I might call this "common path" (which is what [`NSFileManager`](https://developer.apple.com/documentation/foundation/nsfilemanager/1407693-urlfordirectory) call this root.

... so maybe `makeCommonPath`?

Reply:
Thanks for the suggestions! I've renamed the function to `makeCommonPath` to better align with NSFileManager's terminology.